charts/admission-controller/values.yaml

```diff
@@ -152,6 +163,24 @@ podMonitors:
     annotations: {}
 
 webhook:
+  v2:
+    # Number of seconds for the request to time out
+    timeoutSeconds: 5
+    nats:
+      # Allow insecure TLS certificates in backend connection to NATS service
+      insecure: false
+      # Override the NATS service connection URL
+      url: ""
+    service:
+      # Use this type as webhook service
+      type: ClusterIP
+      # Configure port for the V2 webhook service
+      port: 7443
+    http:
+      # HTTP serve port where the requests will be served from
+      port: 7443
+      # KSPM Admission Controller container image
+    image: quay.io/sysdig/secure-admission-controller:1.25.5-rc
```

Comment:
For the other container images we have we separate the registry, repository, and tag values to be independently configurable. Can we adopt that methodology here?

As examples:
- Agent: https://github.com/sysdiglabs/charts/blob/master/charts/agent/values.yaml#L57-L59
- Admission Controller: https://github.com/sysdiglabs/charts/blob/master/charts/admission-controller/values.yaml#L162-L168
- KSPM Collector: https://github.com/sysdiglabs/charts/blob/master/charts/kspm-collector/values.yaml#L77-L82

Reply:
Created issue for tracking: SSPROD-28668.

Similar to https://github.com/sysdiglabs/charts/pull/1214#discussion_r1287404928, should we block the Controlled Availability release and prioritize the ticket, or can we release CA and address this later?

Reply:
@airadier I guess so...otherwise we'll create a BC

Reply:
This should be addressed now.